docs/sdkx-react-native/getting-started.mdx

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 10
+title: Getting Started
+description: "You're 3 steps away from adding great in-app support to your React native application."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Getting Started {#getting-started}
+
+<Intro>
+
+You're 3 steps away from adding great in-app support to your React Native application.
+
+</Intro>
+
+Guide to integrating the React Native plugin for the Helpshift SDK X. You can also refer `helpshift-plugin-sdkx-react-native` package [here](https://www.npmjs.com/package/helpshift-plugin-sdkx-react-native).
+
+## Requirements {#version-requirements}
+
+- Node v16 or above
+- Android Studio
+- Xcode 13 or above
+- Java 11 or above
```

Comment:
Do we really need java 11 ?

Reply:
In react native official document it is recommended to have java 11 
https://reactnative.dev/docs/environment-setup#:~:text=React%20Native%20also%20requires%20Java,installed%20using%20Chocolatey%20as%20well.&text=lts%20microsoft%2Dopenjdk11-,If%20you%20have%20already%20installed%20Node%20on%20your%20system%2C%20make,problems%20using%20higher%20JDK%20versions.

Reply:
https://reactnative.dev/docs/environment-setup#xcode Xcode can be 10+.

Reply:
In java, they recommend using java 11 or below not the latest, I don't think we can say minimum is java 11.